src/main/java/uk/gov/hmcts/reform/sscs/services/ccd/UpdateCcdAppellantData.java

```diff
@@ -128,44 +140,20 @@ private boolean updateCcdAppellantContact(Appellant gapsAppellant, Appellant exi
         return false;
     }
 
-    private boolean updateCcdAppellantName(Appellant gapsAppellant, Appellant existingCcdAppellant) {
-        Name gapsAppellantName = gapsAppellant.getName();
-        Name existingCcdAppellantName = existingCcdAppellant.getName();
-
-        if (null == existingCcdAppellantName) {
-            existingCcdAppellant.setName(gapsAppellantName);
-            return true;
-        }
-        boolean dataChanged = false;
-        if (null != gapsAppellantName) {
-            if (StringUtils.isNotBlank(gapsAppellantName.getFirstName())
-                && !gapsAppellantName.getFirstName().equals(existingCcdAppellantName.getFirstName())) {
-                existingCcdAppellantName.setFirstName(gapsAppellantName.getFirstName());
-                dataChanged = true;
-            }
-            if (StringUtils.isNotBlank(gapsAppellantName.getLastName())
-                && !gapsAppellantName.getLastName().equals(existingCcdAppellantName.getLastName())) {
-                existingCcdAppellantName.setLastName(gapsAppellantName.getLastName());
-                dataChanged = true;
-            }
-        }
-        return dataChanged;
-    }
-
     private boolean updateCcdAppointee(Appellant gapsAppellant, Appellant existingCcdAppellant) {
         if (null == gapsAppellant.getAppointee()) {
             return false;
         }
 
         if (null == existingCcdAppellant.getAppointee()) {
-            existingCcdAppellant.setAppointee(gapsAppellant.getAppointee());
+            existingCcdAppellant.setAppointee(Appointee.builder()
```

Comment:
why we have to build the Appointee here using Appointee.builder()? I don't have to change this line

Reply:
I have to copy all the elements except for Name.

Reply:
Here, it's creating new Appointee object without name, which will will override the existing Appointee

Reply:
It is as per requirement